extension/bearertokenextension/README.md

```diff
@@ -0,0 +1,40 @@
+# Authenticator - Bearer
+
+This extension implements `configauth.Authenticator` and is to be used in grpc receivers inside the `auth` settings as a means
+to embed a static token for every rpc call that will be made.
+
+The authenticator type has to be set to `bearer`.
+
+## Configuration
+
+```yaml
+extensions:
+  bearer:
```

Comment:
Do we believe authentication extensions should have a naming convention? Example "bearer_token_auth"? I'm torn on the need for a naming convention, but not sure it's obvious for those that may not know certain security terms. Thoughts @jpkrohling?

Reply:
Also, the name of the directory is "bearertokenextension" while the configuration name is "bearer". Whatever we agree upon for the name, my opinion is that these should be the same if possible/makes sense.

Reply:
Yes, I think it will be useful to use `auth` suffix for all authenticator extensions. And, yes the directory names of components typically follow the `name+kind` pattern.